cpp/arcticdb/storage/azure/azure_mock_client.cpp

```diff
@@ -30,13 +30,16 @@ std::string operation_to_string(AzureOperation operation){
 std::string MockAzureClient::get_failure_trigger(
         const std::string& blob_name,
         AzureOperation operation_to_fail,
+        std::string error_code,
```

Comment:
`const std::string&`?

Reply:
Fixed in [4ea3442](https://github.com/man-group/ArcticDB/pull/1344/commits/4ea3442882bfdc4001ff117531804a8ba28a8d08)